src/main/zlatopolsky/chapter10/Ch1N10_43.java

```diff
@@ -0,0 +1,18 @@
+package zlatopolsky.chapter10;
+
+public class Ch1N10_43 {
+
+    public static int sumDigits(int x) {
+        if (x <= 0) {
+            return x;
+        }
+        return x % 10 + (sumDigits(x / 10));
+    }
+
+    public static int sumDigits(int x, int count) {
+        if (x <= 0) {
```

Comment:
//code duplication, int count - you didn't use this variable.

Reply:
yes, fix error and rename method